src/operations/executors/string_executor_test.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * @license
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =============================================================================
+ */
+import * as tfc from '@tensorflow/tfjs-core';
+
+import {ExecutionContext} from '../../executor/execution_context';
+import {Node} from '../types';
+
+import {executeOp} from './string_executor';
+// tslint:disable-next-line:max-line-length
```

Comment:
this is not needed.

Reply:
removed (https://github.com/tensorflow/tfjs-converter/pull/376/commits/c40b65c9ec9e71f5f5c66562540ae95a74decc8d)